src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -442,11 +465,15 @@ def map_entries(self, entry):
         ElasticSearch
         :return: A HitEntry Object with the appropriate fields mapped
         """
+        file_type_summaries = self.make_file_type_summaries(self.make_files(entry))
 
         return HitEntry(
             processes=self.make_processes(entry),
             entryId=entry["entity_id"],
-            files=self.make_files(entry),
+            fileTypeSummaries=[FileTypeSummary.create_object(
+                file_type=file_type,
```

Comment:
Aligned indent …

Reply:
You mean align like this?
```
            fileTypeSummaries=[FileTypeSummary.create_object(file_type=file_type,
                                                             total_size=file_type_summary['size'],
                                                             count=file_type_summary['count'])
                               for file_type, file_type_summary in file_type_summaries.items()],
            projects=self.make_projects(entry),
```
---
Is this about convention or readability? I personally find more readable example below.
```
           fileTypeSummaries=[FileTypeSummary.create_object(
                file_type=file_type,
                total_size=file_type_summary['size'],
                count=file_type_summary['count'])
                for file_type, file_type_summary in file_type_summaries.items()],
            projects=self.make_projects(entry),
```